### PR TITLE
BF: If n_frames is larger than one use path-numbering.

### PR DESCRIPTION
**Pull request by arokem**
Otherwise, each consecutive frame over-writes the previous one.

### CODE REVIEW

**Comment by stefanv** [comment]
Looks good!  Want to also fix the spelling error in the ``path_numering`` parameter description?

**Comment by arokem** [comment]
Thanks for the feedback. Added another commit fixing up the docstring.

File: dipy/viz/fvtk.py [hunk 1]
```diff
@@ -1753,29 +1753,30 @@ def record(ren=None, cam_pos=None, cam_focal=None, cam_view=None,
...
+        As returned from :func:`ren`.
     cam_pos : None or sequence (3,), optional
-        camera position
+        Camera position
     cam_focal : None or sequence (3,), optional
-        camera focal point
+        Camera focal point
     cam_view : None or sequence (3,), optional
-        camera view up
+        Camera view up. Default
```
Comment (stefanv) [inline dipy/viz/fvtk.py]: Default is

**Comment by stefanv** [comment]
Nice, you caught a lot of fixes!  Just the default of ``cam_view``, then I can merge for you.

**Comment by arokem** [comment]
Yeah - it's whatever the default is when you start one of these devils.
Just removed the word 'default' instead.

**Comment by stefanv** [comment]
:heart: that commit message

**Pull request merged**